package.json

```diff
@@ -24,11 +24,11 @@
     "homepage": "https://github.com/yahoo/jsx-test",
     "devDependencies": {
         "babel": "^5.0",
-        "istanbul": "^0.3.15",
         "jenkins-mocha": "^2.2.0",
         "react": "^0.14.0"
     },
     "dependencies": {
+        "istanbul": "^0.3.15",
```

Comment:
do we really need the istanbul dependency? Based on https://github.com/yahoo/jsx-test/issues/33 i was thinking about removing the dependency of istanbul and babel.